mainnet-v1/sentry/validator/bor/config.toml

```diff
@@ -1,42 +1,132 @@
+# NOTE: Update and uncomment: `keystore`, `password`, and `unlock` fields.
+
 chain = "mainnet"
+# identity = "Pratiks-MacBook-Pro.local"
+# log-level = "INFO"
 datadir = "/var/lib/bor/data"
 # keystore = "$BOR_DIR/keystore"
 syncmode = "full"
+# gcmode = "full"
+# snapshot = true
+# ethstats = ""
 
-[accounts]
-allow-insecure-unlock = true
-# password = "$BOR_DIR/password.txt"
-# unlock = ["$ADDRESS"]
+# [requiredblocks]
 
-[jsonrpc]
-ipcpath = "/var/lib/bor/bor.ipc"
+[p2p]
+    maxpeers = 1
```

Comment:
Do we need just one max peer for validator node?

Reply:
I think this is fine. A validator should connect only to its sentry.

Reply:
Alright.